README.md

```diff
@@ -33,8 +33,76 @@ Exemple de fiche résumée :
 
 Exemple de détail pour un test (historique) :
 ![resultats_single](./documentation/single.png)
+# Ludwig en 5 minutes
 
-# Installation
+Pour utiliser Ludwig avec son application, il faut : 
+
+* Intégrer le widget sur son application et le configurer
+* Installer la partie serveur de ludwig
+
+Les deux composants vont devoir communiquer, il faut donc que le serveur Ludwig soit joignable par le widget.
+
+Tout ce dont va avoir besoin l'application est détaillé dans les exemples et dans le script `setupNPMVariables.sh`.
+
+## Intégrer le widget
+Dans le dépôt, le widget "prêt à servir" est disponible ici : `dist/ludwig.js`, il suffit de l'intégrer dans son application avec un tag `script` pour qu'il soit disponible dans `window.ludwig`. 
+
+`window.ludwig` donne accès au constructeur car il faut configurer le widget.
+
+Exemple d'import et de configuration : 
+
+```html 
+<script type="text/javascript" src="http://url.serveur.ludwig/ludwig.js" charset="utf-8">
+</script>
+<script>
+    var configuration = {
+        repo: 'monutilisateur/nom-depot',
+        branch: 'foo',
+        template: 'un+template+basique\r\n===========',
+        prefix: 'test-ludwig',
+        ludwigCreateSuggestionURL: 'http://url.serveur.ludwig/createSuggestion'
```

Comment:
Are all this options necessary for the *most basic example* ? Could we juste assume `branch = master` and `prefix = '' ?
At this stage, I imagine the `repo` is where my test files will be stored, but it's not clear for me what the `template` is.